lib/blocks.js

```diff
@@ -949,6 +973,27 @@ var findNextNonspace = function() {
     this.indented = this.indent >= CODE_INDENT;
 };
 
+function measureNonspaceColumn(line) {
```

Comment:
I could've extracted this from the above method, but I'm hesitant about modifying the existing functions as much as possible to avoid future conflicts, so I just copied this out